crates/ark/src/data_explorer/histogram.rs

```diff
@@ -606,4 +607,29 @@ mod tests {
             );
         })
     }
+
+    #[test]
+    fn test_frequency_table_haven_labelled() {
+        r_task(|| {
+            if !package_is_installed("haven") {
```

Comment:
Also need to add haven to CI: https://github.com/posit-dev/ark/blob/5c3e1888ca1d864b761b8aaba573ebe99c3c411b/.github/workflows/test-linux.yml#L54